api/bundle/client.go

```diff
@@ -34,7 +34,7 @@ func NewClient(st base.APICallCloser) *Client {
 func (c *Client) ExportBundle() (string, error) {
 	var result params.StringResult
 	if bestVer := c.BestAPIVersion(); bestVer < 2 {
-		return "", errors.Errorf("ExportBundle() on v%d is not supported", bestVer)
+		return "", errors.Errorf("this controller version v%d doen not support bundle feature.", bestVer)
```

Comment:
This output a v1 in the QA work. That's not the controller version so this turns into a bit of misinformation doesn't it?